App.js

```diff
@@ -1,23 +1,22 @@
-import { createAppContainer } from 'react-navigation'
-import { createStackNavigator } from 'react-navigation-stack'
+import 'react-native-gesture-handler'
+import * as React from 'react'
+import { NavigationContainer } from '@react-navigation/native'
+import { createStackNavigator } from '@react-navigation/stack'
 import SearchScreen from './src/screens/SearchScreen'
 import MoreInfoScreen from './src/screens/MoreInfoScreen'
 
 
-const navigator = createStackNavigator(
-{
-  Search: SearchScreen,
-  MoreInfo: MoreInfoScreen,
-}, 
-{
-  initialRouteName: 'Search',
-  defaultNavigationOptions: {
-    title: 'FoodFind',
-    headerStyle: {
-      backgroundColor: '#29434e',
-    },
-    headerTintColor: '#fff',
-  }
-})
+const Stack = createStackNavigator()
 
-export default createAppContainer(navigator) 
\ No newline at end of file
+const App = () => {
+  return (
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName="Search">
+        <Stack.Screen name="Search" component={SearchScreen} options={{title: 'foodFind'}} />
+        <Stack.Screen name="MoreInfo" component={MoreInfoScreen} options={{title: 'RestaurantDetails'}} />
+      </Stack.Navigator>
+    </NavigationContainer>
+  )
+}
+
```

Comment:
Rebuild app.js file with navigation container and stack navigator in a more declarative style.